packages/webapp/components/layouts/ProfileLayout/index.tsx

```diff
@@ -62,8 +62,27 @@ export default function ProfileLayout({
   ogImageUrl.searchParams.set('type', 'wide');
   ogImageUrl.searchParams.set('r', Math.random().toString(36).substring(2, 5));
 
+  const getSeoTitle = () => {
```

Comment:
We could also move this to the individual pages rather maybe?
then you don't need the switch statements here?